frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -93,17 +94,14 @@ const operatorHubFilterGroups = [
   'infraFeatures',
 ];
 
-// t('olm~Source')
-// t('olm~Provider')
-// t('olm~Install state')
-// t('olm~Capability level')
-// t('olm~Infrastructure features')
-const operatorHubFilterMap = {
-  catalogSourceDisplayName: 'Source',
-  provider: 'Provider',
-  installState: 'Install state',
-  capabilityLevel: 'Capability level',
-  infraFeatures: 'Infrastructure features',
+const getOperatorHubFilterMap = () => {
```

Comment:
If this is only used in one component, it's probably simpler just to move the object into that component.

Reply:
@spadgett done!